index.js

```diff
@@ -185,7 +195,7 @@ function getManifestWithUpdatedVoltaObject () {
   });
 }
 
-async function run () {
+async function run() {
```

Comment:
I don't know how this isn't failing the linter?

Only have the parens touching the function name if it is executing the function:

* `run()`

Otherwise, like when defining a function, have a space between to clearly visually communicate the intent of the code at a glance.

* `run ()`

Reply:
Gotcha

Reply:
I assume you are using "Prettier", which is known for conflicting with all other tools, including itself. They even have a warning on their own site about this. Due to the lack of configurability, the very very poor default choices it comes with, and the fact that it can't play nice with any other tools I have avoided its use in favor of EditorConfig for automated formatting of basic things (EOL, indentation, Line Endings), and ESLint for everything else. Between those two it covers everything Prettier attempts to do with better control and without the downsides of breaking other tooling.

Reply:
I actually don't. But my editor config is probably the culprit.

Reply:
In general I try to keep the diff as small as possible since I know my editor settings won't match the project's settings 100%